src/irmin-mirage/git/irmin_mirage_git_intf.ml

```diff
@@ -67,7 +67,7 @@ module type KV_RO = sig
     ?branch:string ->
     ?root:key ->
     ?ctx:Mimic.ctx ->
-    ?headers:Cohttp.Header.t ->
+    ?headers:(string * string) list ->
```

Comment:
it's a bit of a shame to loose type information here

Reply:
I agree that this is sad, but removing the conduit & cohttp dependency cone (and enable us to use httpaf) is in my opinion a big advantage.

Of course, there could be a common http signature (maybe mirage-http), but I'm not convinced that it is worth developing this.

Reply:
Same opinion as @hannesm, the best will be `mirage-http` indeed (and `ocaml-letsencrypt` started to provide its own required interface) but the underlying work is too huge to consider it in the short-term.